app/transport/rest/contracts.go

```diff
@@ -7,5 +7,6 @@ import (
 // UserUsecase represents User use-case layer.
 type UserUsecase interface {
 	SignUp(entities.User) (string, error)
-	GetUser(id string) (*entities.User, error)
+	Fetch(string, string, string) ([]entities.User, error)
```

Comment:
Not required: GetAll is a more common name in such a case

Reply:
Noted